website/content/en/docs/scorecard-alpha/_index.md

```diff
@@ -0,0 +1,222 @@
+test is defined within the test itself
+ * image - the test container image name that implements a test 
+ * entrypoint - the command and arguments that are invoked in the 
+test image to execute a test
+ * labels - user defined labels that allow for test selection using 
+the scorecard CLI `--selector` flag
+ * description - the human readable description for the test, however, 
+the description printed by scorecard on an executed test is the 
+description provided by the test implementation itself
+
+### Command Args
+
+The scorecard command has the following syntax:
+```
+operator-sdk alpha scorecard [bundle path] | [bundle image name] [flags]
+```
+
+The scorecard requires a positional argument that holds either the
+on-disk path to your operator bundle or the name of a bundle image.
+
+Scorecard flags include the following:
+
+| Flag        | Type   | Description   |
+| --------    | -------- | -------- |
+| `--config`  | string | path to config file (default `<bundle directory>/tests/scorecard/config.yaml`; file type and extension must be `.yaml`). If a config file is not provided and a config file is not found at the default location, the scorecard will exit with an error. |
+| `--kubeconfig`, `-o`  | string |  path to kubeconfig used by the scorecard to execute tests, if not supplied, the KUBECONFIG environment variable or $HOME/.kube/config path is assumed, if those are not found an in-cluster configuraiton is assumed. |
+| `--list`, `-L`  | bool |  if true, only print the test names that would be run based on selector filtering. |
+| `--namespace`, `-n`  | string |  the namespace for scorecard to use when executing test Pods |
+| `--output`, `-o`  | string | output format. Valid options are: `text` and `json`. The default format is `text`, which is designed to be a simpler human readable format.|
+| `--selector`, `-l`  | string |  the label selector to filter tests on. |
+| `--service-account`, `-s`  | string |  the service account for scorecard to use when creating test Pods.|
+| `--skip-cleanup`, `-x`  | boolean |  disable deleting test Pods and ConfigMaps generated from executing scorecard tests, useful for debugging.|
+| `--wait-time`, `-w`  | int |  seconds to wait for tests to complete.  Example is 35s.|
+
+## Tests Performed
+
+Scorecard users can specify a `--selector` CLI flag to filter which 
+tests it will execute.  If a selector flag is not supplied, then all 
+the tests within the scorecard configuration file are executed.
+
+Tests are executed serially, one after the other, with test results 
+being aggregated by scorecard to present back to the end user.
+
+Here are some examples of supplying a selector flag to scorecard:
+```sh
+operator-sdk scorecard -o text --selector=test=basic-check-spec-test
+operator-sdk scorecard -o text --selector=suite=olm
+operator-sdk scorecard -o text --selector='test in (basic-check-spec-test,olm-bundle-validation-test)'
+```
+
```

Comment:
May we would clarify what is happens in each example. Also, would not be better to have a specific section for it, something as "Using selectors"?

Reply:
good idea, updated.